bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/component.jsx

```diff
@@ -257,6 +258,7 @@ class UserParticipants extends Component {
             <TransitionGroup ref={(ref) => { this.refScrollItems = ref; }}>
               { this.getUsers() }
             </TransitionGroup>
+            <div style={matchMedia(`only screen and (max-width: 640px)`).matches ? {margin: '4rem'}:{}}> </div>
```

Comment:
why don't use className? with className, you can use mq() and breakpoints.
E.G.: 
```
YOUR_CLASS_NAME {
	@include mq($small-only) {
		mergin: 4rem;
	}
}  
```

[Breakpoints](https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bigbluebutton-html5/imports/ui/stylesheets/variables/breakpoints.scss#L34)